### PR TITLE
My feature

### PR DESCRIPTION
**Pull request by MarvellousUbani**

### CODE REVIEW

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Hi @MarvellousUbani, @Ashah15 :wave:

Good job working on the project so far. :clap:

- [ ] This project is basically to make sure that you fully understand positioning and floats without using `bootstrap`. It's a good point that you are already know how to use bootstrap, but you will have the time to use bootstrap in upcoming projects. So make sure that this project is done without `bootstrap`
- [ ] Let's make sure that we use [stickler](https://github.com/microverseinc/linters-config/tree/master/css). It's much easier now to use, just follow this link. It's important tool that ensure that our code base has no errors, typos, and have healthy style guidelines.
- [ ] Let's add the footer section & social media icons & copyright as seen in the NYT page.
- [ ] Make sure to complete the `readme.md` [properly](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) with `contributors` information section and `about` section or any information you would find relevant.

Let's make sure that these requirements are handled, then please submit a new code review.
Happy coding :computer:

**Review by bolah2009 — CHANGES_REQUESTED** [review]
# Hi Team :wave: 

Seems this project is still not implemented without using bootstrap as suggested by @mosaaleb 
Fix this and submit a request for another review :+1: 

Feel free to reach out if you have any question.

Keep up the good work and best of luck on the next submission. 👍

Happy Coding!  :dizzy: and Have a Nice Day :smile:

Cheers,  
Bola

File: index.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    <p>“It was a very special moment, and one we took very seriously as scientists,” said Dr. Kovac, who chose his words as carefully as he tended his radio telescopes.</p>
+
+    <p>Andrei Linde of Stanford, a prolific theorist who first described the most popular variant of inflation, known as chaotic inflation, in 1983, was about to go on vacation in the Caribbean last week when Chao-Lin Kuo, a Stanford colleague and a member of Dr. Kovac’s team, knocked on his door with a bottle of Champagne to tell him the news.</p>
+   </div>
+
+
+ </div>
+ </div>
+
+
```
Comment (bolah2009) [inline index.html:361]: `REQUIRED`

The page is currently missing some section. Please check the original site.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+	<title>Space Ripples Reveal Big Bang's Smoking Gun</title>
+</head>
+<body>
+
+  <nav class="top-nav">
+  			<div class="row top-sub-nav">
+  				<div class="col">
+						<button class="trans-btn d-inline"><i class="fa fa-bars pryt1" aria-hidden="true" style="color:#464646;"></i></button>
+            <button class="trans-btn d-inline"><i class="fa fa-search" aria-hidden="true"></i></button>
+            <p class="cat-title d-inline"><strong>SPACE & COSMOS</strong></p>
```
Comment (bolah2009) [inline index.html]: `REQUIRED`

Consider wrapping with an anchor tag (`<a>`) as shown below.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+					
+  				</div>
+
+  				<div class="col logo-col">
+  					<img src="image/newyorktimes.png" class="img-fluid logo" alt="new york times logo">
```
Comment (bolah2009) [inline index.html]: `REQUIRED`

Consider wrapping this logo with an anchor tag (`<a>`).

File: index.html [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  				</div>
+
+  				<div class="col">
+
+  					 		<button class="btn btn-blue btn-login" type="submit">Login</button>
+
```
Comment (bolah2009) [inline index.html]: `REQUIRED`

This section should contain two buttons, see below image.

sub login preffered

File: index.html [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+  				</div>
+
+			</div>
+
+		</nav>
+
+ <div class="middle-section">
+  <div class="row">
+  <h1 class="top-title"><em>Space Ripples Reveal <br> Big Bang's Smoking Gun</em></h1>
+  <img src="image/BigBang.jpg" alt="Smoking gun" class="img-fluid">
```
Comment (bolah2009) [inline index.html]: `REQUIRED`

All images and videos should be enclosed by a `<figure>` tag with `<figcaption>` that gives a description of the embedded image or video

File: index.html [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+
+
+
+
+<!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
```
Comment (bolah2009) [inline index.html]: `SUGGESTION`

It's better to delete redundant code.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
# Nice Job Team! :clap: 

You've done an awesome Job :muscle:

- A few changes to be made, please check the screenshot below and kindly make changes accordingly. Check the [original site](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html) for more details.

| Current | Preffered |
| :---:   | :---:   |
|  btn nyt current | sub login preffered  |
| NOT FOUND   | read button preffered  |
| NOT FOUND  | editor's pick  |
| NOT FOUND  | sign up  |
| NOT FOUND  |  nyt popular preffered |
| NOT FOUND  | nyt aside preffered |
| NOT FOUND | nyt editor preffered  |
| NOT FOUND  | nyt footer preffered  |

You have done an awesome job, consider making these changes, taking into consideration the **positioning and styling** of the above-screenshotted components of the page **including the whole footer section** of the page.

-----

### To make the review even much easier for you, below are the checklist to consider the project `complete`  as it must satisfy the following points:

#### Navigation bar

:heavy_check_mark:   It has a navigation bar

:heavy_check_mark:    It uses the `<nav>` tag

:heavy_check_mark:   It has a "Hamburger button", a search icon and a link with text on the leftmost side of the navbar

:heavy_check_mark:   has the NY Times image logo embedded in the center of the navbar

:recycle:  The NY Times logo is a link

:recycle:   It has both, 'Subscribe' and 'Log in' elements with a `<button>` tag on the rightmost side of navbar

:recycle:    The 'Subscribe' and 'Log in' buttons have a blue background and white letters

#### Bottom
:recycle:   It should have a 'Read comments' button and social media icons (fontawsome is recommended for the icons)

#### Additional Links

:recycle:  The section should be made of two parts: left side for links with images and right for links only

:recycle:    The left side is made of two parts: *More in Space and Cosmos* and *Editor's picks*

#### Footer
:recycle:    It includes the NYTimes logo as an embedded image in the top left corner of the footer

:recycle:  The NYTimes logo is a link

:recycle:  It includes a *go to homepage* link on the top right corner of the footer

:recycle:  The bottom part of the footer to includes links with copyright, TOC, contacts, and help

#### Navigation bar

:recycle:   The Navbar should have a fixed position

:heavy_check_mark:     Floats or flex display for positioning items inside the navbar

#### Main Content
:recycle:      The main content of the website should be wrapped with an HTML tag  `<main>`

:recycle:   The first image should be ~ 50% wider than the rest of the content inside `<main>`

:recycle:   All images and videos should be enclosed by a `<figure>` tag with `<figcaption>` that gives a description of the embedded image or video.

:recycle:   Any sections that include text (paragraphs) should be wrapped with a `<section>` tag 

:recycle:  Include an `<aside>` tag for Editor's picks and a Sign-Up button.

#### Additional Links
:recycle:   The left side uses `<figure>` and `<figcaption>` tags followed by the title of the articles

:heavy_check_mark:    `display: flex` or *float* are recommended

#### Footer
:recycle:     The footer section is wrapped by a `<footer>` tag

:heavy_check_mark:    Links should be organized using the `<ul>` tag and positioned with `display: flex` or *floats*


Keep up the good work and best of luck on the next submission. 👍

Cheers
Bola

File: css/styles.css [hunk 7]
```diff
@@ -0,0 +1,320 @@
...
+.d-inline{
+	display: inline;
+}
+
+.top-nav{
+		border-bottom: 1px  solid #e5e5e5;
+		width: 100%;
+    	position: fixed;
+    	top: 0;
+    	background: #fff;
```
Comment (bolah2009) [inline css/styles.css:24]: Consider adding `z-index` property here as explained in [this comment](https://github.com/MarvellousUbani/NewYorkTimes/pull/1#pullrequestreview-265746079).

**Review by bolah2009 — APPROVED** [review]
# Congratulations Team! :tada:

You've done an awesome Job :muscle: :clap:

#### The [page](https://raw.githack.com/MarvellousUbani/NewYorkTimes/my-feature/index.html) looks very good to me. :clap: (Plus you even add ad banner :open_mouth: ) :+1: 

Check out the code with [HTML Validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2FMarvellousUbani%2FNewYorkTimes%2Fmy-feature%2Findex.html) and **increase/add the `z-index` CSS property of the navbar** to prevent the below image behaviour.

current navbar z-index

For example adding the following to that navbar class (`top-nav`)

``` CSS
.top-nav {
    .
    .
    z-index: 10;
}
```

Will result in the following image:

preffered navbar z-index

#### Learn more about `z-index` [here](https://developer.mozilla.org/en-US/docs/Web/CSS/z-index).

Good luck with your next project
Keep it up :star2:

Cheers
Bola

**Pull request merged**